crates/turbopack-ecmascript-runtime/js/src/dev/runtime/nodejs/runtime-backend-nodejs.ts

```diff
@@ -85,7 +85,6 @@ let BACKEND: RuntimeBackend;
     const resolved = require.resolve(
       "./" + path.relative(path.dirname(fromChunkPath), chunkPath)
     );
-    delete require.cache[resolved];
```

Comment:
This might break `NextMode::DevServer` reloading in SSR.

Reply:
It should be in a fresh process